cpp/cmake/thirdparty/get_nvcomp.cmake

```diff
@@ -13,14 +13,15 @@
 # =============================================================================
 
 # This function finds nvcomp and sets any additional necessary environment variables.
-function(find_and_configure_nvcomp VERSION)
-
-  # Find or install nvcomp
+function(find_and_configure_nvcomp VERSION_MIN VERSION_MAX)
+  # Search for latest version of nvComp
+  rapids_find_package(nvcomp ${VERSION_MAX} QUIET)
+  # If latest isn't found, fall back to building oldest support from source
```

Comment:
If we don't find anything does it make more sense to build the _newest_ supported version? It seems safe and could give us performance improvements or bug fixes that we might not get from the oldest. Is there a reason to prefer oldest?

Reply:
Currently 2.2.0 is the latest version of nvComp which has been mirroed to github. While I agree, we can't currently build 2.3.0 from source as the fallback

Reply:
Got it, that makes sense.